src/cysignals/signals.pyx

```diff
@@ -110,16 +109,25 @@ cdef int sig_raise_exception "sig_raise_exception"(int sig, const char* msg) exc
         if msg is NULL:
             msg = "Floating point exception"
         PyErr_SetString(FloatingPointError, msg)
-    elif sig == SIGBUS:
-        if msg is NULL:
-            msg = "Bus error"
-        PyErr_SetString(SignalError, msg)
     elif sig == SIGSEGV:
         if msg is NULL:
             msg = "Segmentation fault"
         PyErr_SetString(SignalError, msg)
-    else:
-        raise SystemError(f"unknown signal number {sig}")
+    IF UNAME_SYSNAME != 'Windows':
```

Comment:
Do you really need this? In several places (not only here), you have code with cases for various signals. Could you make this more uniform by doing something like
```
#ifndef SIGBUS
#define SIGBUS -1   /* unused */
#endif
```